### ISSUE TITLE
[Suggestion] Don't put "New Tab" into the map

### ISSUE THREAD

**Issue by jeff9315** [GarinZ/link-map#71]
I've just started using Link Map. One suggestion is that Chrome's "New Tab" appears in every browser window. Maybe other browsers have it named differently. Have a blacklist of tabs that are NOT kept in Link Tab.

**Comment by dveb8886** [comment]
I disagree with this. It's preferable for all tabs to be tracked, including new tabs. At most, this should be configurable

**Comment by tmcpeck** [comment]
I agree also with dveb8886 and would like to request (if I am blind I will admit to it) (and will start another [Suggestion] if needed) that a "New Tab" button be put into Link Map.  I have seen these like a Plus symbol at the bottom of some other Vertical Tab extension such as Thready.  Right now all I can find after right-clicking and checking every setting in Link Map, I still have to go to the end of Chromes horizontal tabs to hit the plus to add another "New Tab" to my bring up my new tab page.

Other than that pretty impressed with Link Map already.

![image](https://github.com/user-attachments/assets/c7bd69f9-b7d9-432c-a5c5-0ce824ffa9b5)
